app/selectors/preferences.js

```diff
@@ -46,8 +46,10 @@ export const getTheme = createSelector(
         // At this point, the theme should be a plain object
 
         // Fix a case where upper case theme colours are rendered as black
-        for (const key of Object.keys(theme)) {
-            theme[key] = theme[key].toLowerCase();
+        if (currentTeamId) {
```

Comment:
added this check here cause when logging out from **no teams** it was giving an error of mutability